### DISCUSSION TITLE
how to change ERC-20 to my own DRC-20 ?

### DISCUSSION THREAD

**Discussion by noorshanu** [blockscout/blockscout#8718] [Support]
in ui everywhere its ERC-20 i want to change it.how can i change it without error any idea..actually i want to change from ERC-20 to Drc-20 in my frontend but everytime i change it .it gives 500 error

**Comment by Mojmir-R** [comment]
You will need to provide a much more detailed inquiry in order for anyone to be able to advise you here.
